nifi-extension-bundles/nifi-protobuf-bundle/nifi-protobuf-services/src/test/java/org/apache/nifi/services/protobuf/TestStandardProtobufReaderPropertyValidation.java

```diff
@@ -84,6 +84,62 @@ void testValidWhenSchemaTextPropertyRemoved() { // default value of ${proto.sche
             enableAllControllerServices();
             runner.assertValid(standardProtobufReader);
         }
+
+        @Test
+        void testInvalidSchemaTextFormat() {
+            runner.setProperty(standardProtobufReader, SCHEMA_TEXT, "invalid protobuf schema");
+            final ValidationResult invalidResult = verifyExactlyOneValidationError();
+
+            assertTrue(invalidResult.getExplanation().contains("Invalid protobuf schema format"));
+        }
+
+        @Test
+        void testInvalidMessageNameInSchema() {
+            final String validProtobufSchema = """
+                syntax = "proto3";
+                package test;
+                message Person {
+                  string name = 1;
+                  int32 age = 2;
+                }
+                """;
+
+            runner.setProperty(standardProtobufReader, SCHEMA_TEXT, validProtobufSchema);
+            runner.setProperty(standardProtobufReader, MESSAGE_NAME_RESOLUTION_STRATEGY, MESSAGE_NAME_PROPERTY);
+            runner.setProperty(standardProtobufReader, MESSAGE_NAME, "test.NonExistentMessage");
+
+            final ValidationResult invalidResult = verifyExactlyOneValidationError();
+
+            assertTrue(invalidResult.getExplanation().contains("Message name 'test.NonExistentMessage' cannot be found"));
```

Comment:
Recommend reducing the search string:
```suggestion
            assertTrue(invalidResult.getExplanation().contains("test.NonExistentMessage"));
```